components/ai_chat/core/browser/conversation_driver.h

```diff
@@ -200,6 +201,27 @@ class ConversationDriver : public ModelService::Observer {
 
   virtual void OnFaviconImageDataChanged();
 
+  using FetchSearchQuerySummaryCallback = base::OnceCallback<void(
+      const std::optional<SearchQuerySummary>& search_query_summary)>;
+
+  // Get summarizer-key meta tag content from Brave Search SERP if exists and
+  // use it to fetch search query and summary from Brave search chatllm
+  // endpoint.
+  virtual void FetchSearchQuerySummary(
```

Comment:
Should this be `protected`

Reply:
What do you mean? It's under `protected` section.

Reply:
:facepalm: